.github/workflows/Documentation.yml

```diff
@@ -48,10 +48,16 @@ jobs:
           version: "min"
       - uses: julia-actions/cache@v2
         id: julia-cache
+      - uses: actions/create-github-app-token@v2
+        id: generate_token
+        if: ${{ ! github.event.pull_request.head.repo.fork }}
+        with:
+          app-id: "${{ secrets.NUMTERRA_BOT_ID }}"
+          private-key: "${{ secrets.NUMTERRA_BOT_PRIVATE_KEY }}"
       - uses: julia-actions/julia-docdeploy@v1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }}
+          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
```

Comment:
On a second thought, I'm not entirely sure an app is needed for this, pushing to the other repo happens via `DOCUMENTER_KEY`.  Let me try that.